### PR TITLE
Document GDScript format strings

### PR DESCRIPTION
**Pull request by eska014**
Also add missing word "of" in the GDScript reference

### CODE REVIEW

**Comment by akien-mga** [comment]
Awesome! :)

File: reference/gdscript_printf.rst [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+| ``c`` | A single **Unicode character**. Expects an unsigned 8-bit integer   |
+|       | (0-255) for a code point or a single-character string.              |
++-------+---------------------------------------------------------------------+
+| ``d`` | A **decimal integral** number. Expects an integral or real number   |
+|       | (will be floored).                                                  |
++-------+---------------------------------------------------------------------+
+| ``o`` | An **octal integral** number. Expects an integral or real number    |
+|       | (will be floored).                                                  |
++-------+---------------------------------------------------------------------+
+| ``x`` | A **hexadecimal integral** number with **lower-case** letters.      |
```
Comment (bojidar-bg) [inline reference/gdscript_printf.rst]: `integer` you mean?

Comment (eska014) [inline reference/gdscript_printf.rst]: It's an adjective: 'an integral number'. The modifiers (the `+` character) refer to 'number specifiers'.

Comment (bojidar-bg) [inline reference/gdscript_printf.rst]: The thing is that `int` stands for integer. Also, the two are rather different -- https://en.wikipedia.org/wiki/Integer vs https://en.wikipedia.org/wiki/Integral

Comment (eska014) [inline reference/gdscript_printf.rst]: An "integral" is a noun, "integral" in "integral numer" is an adjective, meaning:

> being [...] one or more mathematical integers
> http://www.merriam-webster.com/dictionary/integral

**Pull request merged**